src/presets/cloudflare/runtime/cloudflare-module.ts

```diff
@@ -96,6 +96,26 @@ export default {
       );
     }
   },
+  email(event: any, env: CFModuleEnv, context: ExecutionContext) {
+    if (import.meta._tasks) {
+      (globalThis as any).__env__ = env;
+
+      const domain = (event.to as string).split("@")[1];
+
+      context.waitUntil(
+        runEmailTask(domain, {
```

Comment:
Here, we shall simply call `event.waitUntil(nitroApp.hooks.callHook('cloudflare:email', event, { env, context })`

Reply:
Is it right "cloudflare:email" hooks on here?

https://github.com/unjs/nitro/blob/40a0007a65440187d236efdeb1403c8c301bafe1/src/runtime/types.ts#L24